Bundle/Controller/OpenApi/SubscribersController.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace KwcNewsletter\Bundle\Controller\OpenApi;
+
+use KwcNewsletter\Bundle\Controller\SubscribersApiController;
+use FOS\RestBundle\Controller\Annotations\QueryParam;
+use FOS\RestBundle\Controller\Annotations\RequestParam;
+use FOS\RestBundle\Controller\Annotations\View;
+use FOS\RestBundle\Request\ParamFetcher;
+use KwcNewsletter\Bundle\Model\Subscribers;
+use KwfBundle\Validator\ErrorCollectValidator;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
+use Symfony\Component\Validator\Constraints\Country;
+use Symfony\Component\Validator\Constraints\Email;
+use Symfony\Component\Validator\Constraints\Ip;
+
+/**
+ * @Route("/api/v1/open", service="kwc_newsletter.controller.open_api.subscribers")
+ */
+class SubscribersController extends SubscribersApiController
```

Comment:
extending another controller is not a good pattern to use, instead business logic could be in a service used by both controllers